airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcEncryption.kt

```diff
@@ -0,0 +1,193 @@
+                        null
+                    )
+                clientCertKeyStorePair = Pair(clientCertKeyStoreUri, clientKeyPassword)
+            } catch (ex: Exception) {
+                when (ex) {
+                    is CertificateException,
+                    is IOException,
+                    is KeyStoreException,
+                    is NoSuchAlgorithmException,
+                    is InvalidKeySpecException,
+                    is InterruptedException -> {
+                        throw RuntimeException(
+                            "Failed to create keystore for Client certificate",
+                            ex
+                        )
+                    }
+                }
+            }
+        }
+        return clientCertKeyStorePair
+    }
+
+    fun parseSSLConfig(): Map<String, String> {
+        var caCertKeyStorePair: Pair<URI, String>?
+        var clientCertKeyStorePair: Pair<URI, String>?
+        val additionalParameters: MutableMap<String, String> = HashMap()
+
+        if (!sslMode.isNullOrEmpty()) {
+            additionalParameters[SSL_MODE] = sslMode
+
+            caCertKeyStorePair = prepareCACertificateKeyStore()
+
+            if (null != caCertKeyStorePair) {
+                log.debug { "uri for ca cert keystore: ${caCertKeyStorePair.first}" }
+                try {
+                    additionalParameters.putAll(
+                        java.util.Map.of(
+                            TRUST_KEY_STORE_URL,
+                            caCertKeyStorePair.first.toURL().toString(),
+                            TRUST_KEY_STORE_PASS,
+                            caCertKeyStorePair.second,
+                            TRUST_KEY_STORE_TYPE,
+                            KEY_STORE_TYPE_PKCS12
+                        )
+                    )
+                } catch (e: MalformedURLException) {
+                    throw RuntimeException("Unable to get a URL for trust key store")
+                }
+            }
+
+            clientCertKeyStorePair = prepareClientCertificateKeyStore()
+
+            if (null != clientCertKeyStorePair) {
+                log.debug {
+                    "uri for client cert keystore: ${clientCertKeyStorePair.first} / ${clientCertKeyStorePair.second}"
+                }
+                try {
+                    additionalParameters.putAll(
+                        mapOf(
+                            CLIENT_KEY_STORE_URL to clientCertKeyStorePair.first.toURL().toString(),
+                            CLIENT_KEY_STORE_PASS to clientCertKeyStorePair.second,
+                            CLIENT_KEY_STORE_TYPE to KEY_STORE_TYPE_PKCS12
+                        )
+                    )
+                } catch (e: MalformedURLException) {
+                    throw RuntimeException("Unable to get a URL for client key store")
+                }
+            }
+        }
+
+        log.debug { "additional params: $additionalParameters" }
```

Comment:
Should we log this at the `info` level instead? The log line is probably going to get masked anyway. In any case `additional params` is not a very informative log message.

Reply:
I'd worry those are sensitive information (key/certs) though. I'll remove it